libipl/libipl.H

```diff
@@ -32,7 +34,9 @@ enum ipl_error_type {
 	IPL_ERR_FSI_REG,
 	IPL_ERR_PIB_TGT_NOT_FOUND,
 	IPL_ERR_PLAT,
-	IPL_ERR_ATTR_READ_FAIL
+	IPL_ERR_CLK,
```

Comment:
Why you are removing existing reason code  IPL_ERR_ATTR_READ_FAIL?

Reply:
Do we need seperate error code for ATTR_READ and ATTR_WRITE?

Reply:
yes . Not a good idea to remove existing reason code , unless there is strong use case for this , this can impact user applications. in this case read/ write failure are diffrent.

Reply:
Done